phpBB/includes/acp/acp_search.php

```diff
@@ -341,12 +341,12 @@ function index($id, $mode)
 								$this->search->index_remove($ids, $posters, $forum_ids);
 							}
 
-							$post_counter += $this->batch_size;
+							$post_counter = $ids[count($ids) - 1];
```

Comment:
This needs to move into the `if` statement above.

Reply:
Done.